### PR TITLE
Feature/php 8 support

### PR DESCRIPTION
**Pull request by anubisthejackle**
Fixes some issues found with PHP 8 support.

### CODE REVIEW

File: php/class-photonfill.php [hunk 1]
```diff
@@ -1001,7 +1001,7 @@ public function get_lazyload_image( $attachment_id, $size = 'full', $attr = arra
 		 * @param array  $attr Image attributes.
 		 * @return string Picture markup.
 		 */
-		public function get_attachment_picture( $attachment_id, $size = 'full', $attr ) {
+		public function get_attachment_picture( $attachment_id, $size = 'full', $attr = array() ) {
```
Comment (joemcgill) [inline php/class-photonfill.php:1004]: Does this project not use the short array syntax?

**Review by joemcgill — APPROVED** [review]
👍🏻 Looks good to me. The missing end parenthesis is a particularly odd bug that is nice to have fixed. I'm sure this project predated our Alley PHPCS ruleset, and I think I'd generally lean toward deprecating the use of this plugin in our projects as its use case is no longer needed since all supported browsers have implemented responsive images now, so I wouldn't spend the effort to update this.

File: php/class-photonfill.php (hunk 1 above)
Comment (anubisthejackle) [reply php/class-photonfill.php:1004]: It does not, unfortunately.

**Comment by anubisthejackle** [comment]
@joemcgill Completely agree. These updates are largely to support transitioning a client to PHP 8 with plans to fully remove Photonfill in the near future.

**Pull request merged**